cmd/medius/images/push.go

```diff
@@ -142,9 +142,15 @@ func (b *buildAndPublish) Do(entry *common.Entry, timestamp time.Time) ([]string
 		return nil, b.Ctx.Err()
 	}
 
+	b.Log.Info("Building containerdisk index ...")
+	containerDiskIndex, err := build.ContainerDiskIndex([]v1.Image{containerDisk})
+	if err != nil {
+		return nil, fmt.Errorf("error creating the containerdisk index : %v", err)
+	}
+
 	names := prepareTags(timestamp, b.Options.PublishImagesOptions.TargetRegistry, entry, artifactInfo)
 	for _, name := range names {
-		if err := b.pushImage(containerDisk, name); err != nil {
+		if err := b.pushImageIndex(containerDiskIndex, name); err != nil {
```

Comment:
@0xFelix `pushImage(...)` is no longer used because we won't push single image manifest anymore. Do you want to remove it or keep it?

Reply:
I'll remove it because linter complains on unused `pushImage`:
```
cmd/medius/images/push.go:256:27: func `(*buildAndPublish).pushImage` is unused (unused)
func (b *buildAndPublish) pushImage(containerDisk v1.Image, name string) error {
```